core/src/luigi-config/cluster-management/clusters-storage.js

```diff
@@ -0,0 +1,72 @@
+import i18next from 'i18next';
+import { getActiveCluster, getActiveClusterName } from './cluster-management';
+
+const CLUSTERS_KEY = 'busola.clusters';
+
+let inMemoryClusters = {};
+
+export function load() {
+  return {
+    ...JSON.parse(localStorage.getItem(CLUSTERS_KEY) || '{}'),
+    ...JSON.parse(sessionStorage.getItem(CLUSTERS_KEY) || '{}'),
+    ...inMemoryClusters,
+  };
+}
+
+export function save(clusters, defaultStorage) {
+  clear();
+
+  const localStorageClusters = {};
+  const sessionStorageClusters = {};
+  for (const [clusterName, cluster] of Object.entries(clusters)) {
+    switch (cluster?.config?.storage || defaultStorage) {
+      case 'localStorage':
+        localStorageClusters[clusterName] = cluster;
+        break;
+      case 'sessionStorage':
+        sessionStorageClusters[clusterName] = cluster;
+        break;
+      case 'inMemory':
+        inMemoryClusters[clusterName] = cluster;
+        break;
+      default:
+        console.warn(
+          clusterName,
+          ': Unknown storage type ',
+          cluster?.config?.storage,
+          'saving in localStorage',
```

Comment:
please add translations to this message

Reply:
Yeah, I was thinking about it... do we translate `console` functions? Users normally doesn't look here.

Reply:
Ah, you're right. I didn't notice it's a log...